### PR TITLE
bpo-32046: Update 2to3 when converts operator.isCallable(obj).

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue32046

### CODE REVIEW

**Comment by ilevkivskyi** [comment]
Could you please update the test (see failures) and docs (in ``Doc/library/2to3.rst``)?

**Comment by corona10** [comment]
@ilevkivskyi
Thanks for the review!
I've updated it.

File: Lib/lib2to3/fixes/fix_operator.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 """Fixer for operator functions.
 
-operator.isCallable(obj)       -> hasattr(obj, '__call__')
+operator.isCallable(obj)       -> isinstance(obj, collections.abc.Callable)
```
Comment (merwok) [inline Lib/lib2to3/fixes/fix_operator.py]: Why not use the built-in `callable`?  It’s been re-added to Python 3 a few versions ago.

Comment (ilevkivskyi) [reply Lib/lib2to3/fixes/fix_operator.py]: Since ``callable`` is not available in all 3.x versions, this is probably a safer solution.

**Review by ilevkivskyi — APPROVED** [review]

File: Lib/lib2to3/fixes/fix_operator.py (hunk 1 above)
Comment (merwok) [reply Lib/lib2to3/fixes/fix_operator.py]: It depends if 2to3 is meant to convert code to Python 3 all versions, Python 3 currently maintained branches, or specific Python 3.X that shipped that particular 2to3.  Probably a question for python-dev!

Comment (corona10) [reply Lib/lib2to3/fixes/fix_operator.py]: Should I send this issue to the mailing list?

Comment (merwok) [reply Lib/lib2to3/fixes/fix_operator.py]: I think so, to clarify the compatibility policy of lib2to3.

**Comment by Munyola** [comment]
I thought on this article on low version regulations and can automatically trigger highly on focused platforms when fixed properly.

**Comment by corona10** [comment]
https://groups.google.com/forum/m/#!topic/dev-python/yC7hTgQQ3N8

**Comment by ilevkivskyi** [comment]
@corona10 Could you please update the PR as suggested on python-dev?

**Comment by corona10** [comment]
Sure

**Comment by corona10** [comment]
@ilevkivskyi
Updated!

**Review by merwok — APPROVED** [review]

**Comment by corona10** [comment]
@ilevkivskyi
Can you take a look?

**Review by ilevkivskyi — APPROVED** [review]
@corona10 Thanks! This looks good to me, but I can't merge since I am not a core dev.

**Comment by corona10** [comment]
@serhiy-storchaka Can you take a look?

**Comment by merwok** [comment]
Thanks for the patch!

**Pull request merged**